scss/footer.scss

```diff
@@ -0,0 +1,30 @@
+@import 'colors';
+@import 'sizes';
+
+.footer {
+  background-color: $blue;
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  height: 200px;
+  width: 100%;
+}
+
+.footer-copyright {
+  padding-top: 30px;
+  color: white;
+  display: flex;
+  font-size: $text-small;
+  align-self: flex-end;
+}
+
+.footer-title {
+  padding-top: 25px;
```

Comment:
this should be in the parent since you want the title to be separated from the borders of the blue box.